capstone-project/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -35,7 +35,7 @@
 
 // Enum describing which action should be performed.
 enum Action {
-    CREATE,
```

Comment:
I actually think CREATE is more appropriate here. 

DISPLAY is not an action a backend can handle.